src/repositories/sql/users/update_profile.sql

```diff
@@ -0,0 +1,7 @@
+UPDATE users
+SET
+username = $1,
+contact = $2,
+bio = $3,
+updated_at= (SELECT NOW())
+WHERE user_id=$4;
```

Comment:
- Add indentations for readability
```
UPDATE users
  SET
    x=y
  WHERE ...
```
- You can just write `updated_at = NOW()`
- Make spacing consistent, pick either `x = y` or `x=y`, but not both